DevCycle/DVCClient.swift

```diff
@@ -88,41 +88,54 @@ public class DVCClient {
             return
         }
         self.service = service
-        self.service?.getConfig(user: user, enableEdgeDB: self.enableEdgeDB, extraParams: nil, completion: { [weak self] config, error in
-            guard let self = self else { return }
-            if let error = error {
-                Log.error("Error getting config: \(error)", tags: ["setup"])
-                self.cache = self.cacheService.load()
-            } else {
-                if let config = config {
-                    Log.debug("Config: \(config)", tags: ["setup"])
-                }
-                self.config?.userConfig = config
-                
-                self.handleCachedAnonUserId(user: user)
+        
+        var cachedConfig: UserConfig?
+        if let configCacheTTL = options?.configCacheTTL, let disableConfigCache = options?.disableConfigCache, !disableConfigCache {
+            cachedConfig = cacheService.getConfig(user: user, ttlMs: configCacheTTL)
+        }
+        
+        if cachedConfig != nil {
+            self.config?.userConfig = cachedConfig
+            self.isConfigCached = true
+            Log.debug("Loaded config from cache")
```

Comment:
if we're using the cached config, we should trigger the callbacks associated with a config change (ie. variable.onUpdate)

Reply:
hmm what callback do we have to trigger?

Reply:
I'm not sure exactly, I'll poke through the getConfig code and see if I can find it as well. When `variable.onUpdate` is called a a handler is registered, and we need to find what calls those handlers when the config changes

Reply:
In the `DVCVariable` class it seems `addNotificationObserver()` is called on initialization of the variable and then in `propertyChange()` it handles updating the variables. I don't see anything in getConfig though